<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_interaction</class>
 <widget class="QDialog" name="Dialog_interaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idContactLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>idContact</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idContact"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="contenuInteractionLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contenu Interaction</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="contenu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dateInteractionLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date Interaction</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dateInteractionLineEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="ajouter">
      <property name="text">
       <string>Ajouter Interaction</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="affich">
      <property name="text">
       <string>Afficher Interaction</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="recherche">
      <property name="text">
       <string>Rechrerche Interaction</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="rech"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
